src/main/java/org/openlmis/requisition/domain/Requisition.java

```diff
@@ -267,10 +277,20 @@ public void initiate(RequisitionTemplate template,
     // Secondly, if Proof Of Delivery exists and it is submitted ...
     if (null != proofOfDelivery && proofOfDelivery.isSubmitted()) {
       // .. for each line from the current requisition ...
-      getNonSkippedFullSupplyRequisitionLineItems().forEach(requisitionLine -> {
+      nonSkippedFullSupplyItems = (nonSkippedFullSupplyItems == null)
```

Comment:
I guess we can use here block if statement? No need to assign nonSkippedFullSupplyItems to nonSkippedFullSupplyItems.